---
int16[] breakawaystatus
int16[] breakawaypositions